app/queries/control_rate_query_v2.rb

```diff
@@ -0,0 +1,40 @@
+class ControlRateQueryV2
+  def controlled(maybe_region)
+    region = maybe_region.region
+    ReportingPipeline::PatientStatesPerMonth
+      .where(hypertension: "yes", htn_care_state: "under_care")
+      .where("assigned_#{region.region_type}_region_id" => region.id)
+      .where("months_since_registration >= ?", Reports::REGISTRATION_BUFFER_IN_MONTHS)
+      .where(htn_treatment_outcome_in_last_3_months: :controlled)
+  end
+
+  def controlled_counts(region, range: nil)
+    options = group_date_options(range)
+    controlled(region).group_by_period(:month, :month_date, options).count
+  end
+
+  def uncontrolled(maybe_region)
+    region = maybe_region.region
+    ReportingPipeline::PatientStatesPerMonth
+      .where("assigned_#{region.region.region_type}_region_id" => region.id)
```

Comment:
Is this a typo? `region.region.region_type`?

More reason to keep it in one place 😜